.github/workflows/tests.yml

```diff
@@ -3,6 +3,10 @@
 
 name: Tests
 
+concurrency:
+  group: ${{ github.event.pull_request.id }}
```

Comment:
How does this work for `on: push`?

Reply:
Good callout, I followed the advice for [adding a fallback value](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value).

Reply:
Oh man, I didn't know `||` could be used as fallback operator just like in JS! This is nice.